### PR TITLE
[minor] add unbundledJS option

### PR DESCRIPTION
**Pull request by jchip**

### CODE REVIEW

File: packages/electrode-react-webapp/README.md [hunk 1]
```diff
@@ -7,9 +7,7 @@ a bootstrapping React application.  With support for webpack dev server integrat
 
 ## Installing
 
-```
-npm install electrode-react-webapp --save
-```
+    npm install electrode-react-webapp --save
```
Comment (didi0613) [inline packages/electrode-react-webapp/README.md]:
```bash
npm install electrode-react-webapp --save
```
?

File: packages/electrode-react-webapp/lib/express/index.js [hunk 2]
```diff
@@ -9,37 +9,40 @@ const HTTP_REDIRECT = 302;
...
         }
-        _.each(methods, (method) => {
+        _.each(methods, method => {
           if (method === "*") {
             method = "ALL";
           }
-          app[method.toLowerCase()](path, (request, response) => { //eslint-disable-line
-            const handleStatus = (data) => {
+          app[method.toLowerCase()](path, (request, response) => {
+            //eslint-disable-line
```
Comment (didi0613) [inline packages/electrode-react-webapp/lib/express/index.js]: What does this eslint-disable-line for?

File: packages/electrode-react-webapp/lib/koa/index.js [hunk 3]
```diff
@@ -8,57 +9,58 @@ const koaRouter = require("koa-router")();
...
-    });
-    app.use(koaRouter.routes());
-  })
-  .then(() => next())
-  .catch(next);
+      app.use(koaRouter.routes());
+    })
+    .then(() => next())
+    .catch(next);
 };
```
Comment (didi0613) [inline packages/electrode-react-webapp/lib/koa/index.js]: Need to add `/* eslint-enable */` back at the end of the file?

Comment (jchip) [inline packages/electrode-react-webapp/lib/koa/index.js]: that's to disable a single rule for the entire file.

**Comment by didi0613** [comment]
👍 LGTM

**Pull request merged**